### DISCUSSION TITLE
Initiate a new leafmap without loading the OpenStreetMap as the base map

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#121] [Q&A] [answered]
Is there a way to initiate a new leafmap without loading the OpenStreetMap as the base map? If I use m.add_basemap(basemap="Stamen.TonerLite") to load my base map of choice, the OpenStreetMap is still loaded, and it is covered by my base map of choice. I would have to manually uncheck the OpenStreetMap from the setting to turn it off. Is there a better way to do this? Thanks a lot.

**Comment by giswqs** [comment, marked as answer]
Try this:

```python
m = leafmap.Map()
m.clear_layers()
m.add_basemap("Stamen.TonerLite")
m
```

**Reply by shi093** [reply]
It works! Thanks again. 
I also would like to follow up with my previous question (asked in July) about leafmap looks different in Google Colab. At that time, Colab didn't support ipyleaflet. Now leafmap rendered in Colab works much better, but it still looks different than from my local notebook, especially the information box. When I load a polygon layer onto the map, the information box is supposed to show the data field associated with each polygon when mouse hover over, the information box display works fine on my local computer, but miss the zone_name field on Colab, and the spacing between lines in the information box on Colab is also too large.

**Reply by giswqs** [reply]
This is a known issue. Colab does not yet fully support ipyleaflet. Many ipyleaflet features don't work well in Colab. All custom widgets (e.g., toolbar, infobox) are not working well, and they look super big. I don't have a fix for this at the moment. Try the folium backend with Colab or streamlit https://share.streamlit.io/giswqs/leafmap-apps/app.py

**Reply by stevecwlau** [reply]
Hi Dr. Wu. Really appreciate the tool you've developed. I am using the folium backend you recommended but noticed the 'clear_layers' function isn't included. Any ideas for a workaround or other solutions for this with the folium backend? Thanks!
![Untitled](https://github.com/opengeos/leafmap/assets/91710999/e39e1733-a8cd-4044-86f0-8d4fb956eb21)
